recipes/vs2008_express_vc_python_patch/meta.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: vs2008_express_vc_python_patch
+  version: 1.0
+
+source:
+  url: https://github.com/menpo/condaci/raw/master/vs2008_patch.zip
```

Comment:
It isn't ideal but if necessary I'm not against it. 
Does this whole package need to be put in conda-forge-build-setup? At least, I guess that should be the package which depends on this one?